site/gatsby-site/src/components/submissions/schemas.js

```diff
@@ -76,6 +76,11 @@ export const schema = yup.object().shape({
     then: yup.string().required('*Alleged Harmed Parties is required'),
     otherwise: yup.string().nullable(),
   }),
+  implicated_systems: yup.string().when('incident_ids', {
+    is: (incident_ids) => !incident_ids || incident_ids.length == 0,
+    then: yup.string().required('*Alleged Implicated AI Systems is required'),
```

Comment:
I'm not sure we should make this required. Otherwise we will end up using many "unknown" again. What do you think? Can we defer making this required or is there a problem with that?